packages/reporter/cypress/integration/retries_spec.js

```diff
@@ -0,0 +1,396 @@
+/// <reference types="cypress" />
```

Comment:
remove the duplication, and move to `runner/cypress/integration/retries.ui.spec.js`